<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="7" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="53" started-at="2015-07-20T08:50:12Z" finished-at="2015-07-20T08:50:12Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="53" started-at="2015-07-20T08:50:12Z" finished-at="2015-07-20T08:50:12Z">
      <class name="test.StringTest">
        <test-method status="PASS" signature="testIterateAndRemoveChar()[pri:0, instance:test.StringTest@46f5f779]" name="testIterateAndRemoveChar" duration-ms="16" started-at="2015-07-20T01:50:12Z" description="to test iterative method" finished-at="2015-07-20T01:50:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIterateAndRemoveChar -->
        <test-method status="PASS" signature="testIterateAndRemoveChar_EmptySpacePassed()[pri:0, instance:test.StringTest@46f5f779]" name="testIterateAndRemoveChar_EmptySpacePassed" duration-ms="1" started-at="2015-07-20T01:50:12Z" description="to test iterative method for EmptySpace" finished-at="2015-07-20T01:50:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIterateAndRemoveChar_EmptySpacePassed -->
        <test-method status="FAIL" signature="testIterateAndRemoveChar_Null()[pri:0, instance:test.StringTest@46f5f779]" name="testIterateAndRemoveChar_Null" duration-ms="1" started-at="2015-07-20T01:50:12Z" description="to test iterative method for null" finished-at="2015-07-20T01:50:12Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at stringProblem.StringProblem.iterateAndRemoveChar(StringProblem.java:19)
	at test.StringTest.testIterateAndRemoveChar_Null(StringTest.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIterateAndRemoveChar_Null -->
        <test-method status="PASS" signature="testIterateAndRemoveChar_SpecialCharacter()[pri:0, instance:test.StringTest@46f5f779]" name="testIterateAndRemoveChar_SpecialCharacter" duration-ms="2" started-at="2015-07-20T01:50:12Z" description="to test iterative method for special character" finished-at="2015-07-20T01:50:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testIterateAndRemoveChar_SpecialCharacter -->
        <test-method status="PASS" signature="testOneLineRemoveCharMethod()[pri:0, instance:test.StringTest@46f5f779]" name="testOneLineRemoveCharMethod" duration-ms="2" started-at="2015-07-20T01:50:12Z" description="to test default  String method" finished-at="2015-07-20T01:50:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOneLineRemoveCharMethod -->
        <test-method status="PASS" signature="testOneLineRemoveCharMethod_EmptySpacePassed()[pri:0, instance:test.StringTest@46f5f779]" name="testOneLineRemoveCharMethod_EmptySpacePassed" duration-ms="1" started-at="2015-07-20T01:50:12Z" description="to test iterative method for EmptySpace" finished-at="2015-07-20T01:50:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOneLineRemoveCharMethod_EmptySpacePassed -->
        <test-method status="FAIL" signature="testOneLineRemoveCharMethod_SpecialCharacter()[pri:0, instance:test.StringTest@46f5f779]" name="testOneLineRemoveCharMethod_SpecialCharacter" duration-ms="2" started-at="2015-07-20T01:50:12Z" description="to test oneLine method for special character" finished-at="2015-07-20T01:50:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [this is a te$t $tring] but found [this is a tet tring]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [this is a te$t $tring] but found [this is a tet tring]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at test.StringTest.testOneLineRemoveCharMethod_SpecialCharacter(StringTest.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOneLineRemoveCharMethod_SpecialCharacter -->
      </class> <!-- test.StringTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
